### ISSUE TITLE
Create a skeleton for our confluence documentation

### ISSUE THREAD

**Issue by pahatz** [NBISweden/sda-dev-stack#20]
- [x] Create a skeleton of the team's documentation page in confluence. It should include a good estimate of possible topics and sections that should be included and their content.
- [x] Book a meeting with the team, discuss and decide if any of them are relevant. Restructure it as needed.
- [x] Decide next steps (fill in sections, when to use it).

Next steps:
- Panos works on a prototype structure for the GDI project, filling in as much info as possible
- we review this and decide if we adopt this for all projects
- if yes, we introduce small documentation tasks for each sprint to bring this into shape

**Comment by aaperis** [comment]
We discuss this next Tuesday 11:05

**Comment by pahatz** [comment]
I have added some _per product_ specific sections that I thought should exist in our documentation. https://scilifelab.atlassian.net/wiki/spaces/GS/overview?homepageId=2621341943
I have only some short description context for each, if we decide that they are needed, filling them is going to be a bigger task.
Then there are also a lot of team's resources missing. Some have been added here and there since the adaptation of confluence but still run into missing ones daily. Those are also to be discussed where they should land.
